<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="1" failed="1" skipped="1">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-08-20T03:27:45 IST" name="Default suite" finished-at="2025-08-20T03:27:56 IST" duration-ms="11305">
    <groups>
    </groups>
    <test started-at="2025-08-20T03:27:45 IST" name="Default test" finished-at="2025-08-20T03:27:56 IST" duration-ms="11305">
      <class name="day24.SimpleTests">
        <test-method is-config="true" signature="setup()[pri:0, instance:day24.SimpleTests@77847718]" started-at="2025-08-20T03:27:46 IST" name="setup" finished-at="2025-08-20T03:27:55 IST" duration-ms="9180" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="TestTitle0()[pri:0, instance:day24.SimpleTests@77847718]" started-at="2025-08-20T03:27:55 IST" name="TestTitle0" finished-at="2025-08-20T03:27:55 IST" duration-ms="124" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TestTitle0 -->
        <test-method signature="TestTitle1()[pri:1, instance:day24.SimpleTests@77847718]" started-at="2025-08-20T03:27:55 IST" name="TestTitle1" finished-at="2025-08-20T03:27:55 IST" duration-ms="61" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Automation Testing Practic] but found [Automation Testing Practice]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Automation Testing Practic] but found [Automation Testing Practice]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at day24.SimpleTests.TestTitle1(SimpleTests.java:33)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:565)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TestTitle1 -->
        <test-method signature="TestTitle()[pri:2, instance:day24.SimpleTests@77847718]" started-at="2025-08-20T03:27:55 IST" name="TestTitle" finished-at="2025-08-20T03:27:55 IST" depends-on-methods="day24.SimpleTests.TestTitle1" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method day24.SimpleTests.TestTitle() on instance day24.SimpleTests@77847718 depends on not successfully finished methods 
[day24.SimpleTests.TestTitle1() on instance day24.SimpleTests@77847718]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method day24.SimpleTests.TestTitle() on instance day24.SimpleTests@77847718 depends on not successfully finished methods 
[day24.SimpleTests.TestTitle1() on instance day24.SimpleTests@77847718]
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:159)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TestTitle -->
        <test-method is-config="true" signature="close()[pri:0, instance:day24.SimpleTests@77847718]" started-at="2025-08-20T03:27:55 IST" name="close" finished-at="2025-08-20T03:27:56 IST" duration-ms="845" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
      </class> <!-- day24.SimpleTests -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
